src/components/component-playground.jsx

```diff
@@ -485,28 +485,27 @@ module.exports = React.createClass({
   _getFilteredFixtures() {
     var components = this.props.components;
 
-    return _.reduce(components, function(acc, component, componentName) {
-      var fixtureNames = Object.keys(component.fixtures);
-      var filteredFixtures = _.filter(fixtureNames, function(fixtureName) {
+    return _.reduce(components, function(acc, componentProps, componentName) {
```

Comment:
I wouldn't call the 2nd arg `componentProps` since the props are in the fixtures. For lack of a better idea, I think `component` was fine before.